frontend/src/Explorer/View/Blocks.purs

```diff
@@ -148,11 +156,16 @@ blockRow state (CBlockEntry entry) =
 type BlockColumnProps =
     { label :: String
     , clazz :: String
+    , mRoute :: Maybe Route
     }
 
 blockColumn :: BlockColumnProps -> P.Html Action
 blockColumn props =
-    P.div
+    let tag = case props.mRoute of
+                  Just route -> P.link (toUrl route)
+                  Nothing -> P.div
```

Comment:
@akegalj `fromMaybe P.div (P.link <<< toUrl) props.mRoute` won't work. Could you check it again, pls?